src/test-jdk14/java/com/fasterxml/jackson/databind/failing/RecordBasicsTestFailing.java

```diff
@@ -0,0 +1,47 @@
+package com.fasterxml.jackson.databind.failing;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.BaseMapTest;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.records.RecordBasicsTest;
+
+/**
+ * Tests in this class were moved from {@link RecordBasicsTest}.
+ */
+public class RecordBasicsTestFailing extends BaseMapTest {
+    record EmptyRecord() { }
+
+    record SimpleRecord(int id, String name) { }
+
+    record RecordOfRecord(SimpleRecord record) { }
+
+    record RecordWithIgnore(int id, @JsonIgnore String name) { }
+
+    record RecordWithRename(int id, @JsonProperty("rename")String name) { }
```

Comment:
only SnakeRecord seems to be needed - not the other record definitions

Reply:
Sharp eyes. Patched.